compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/generators/ArgumentsGenerationUtils.kt

```diff
@@ -192,20 +205,22 @@ fun StatementGenerator.generateCallReceiver(
             }
             dispatchReceiverValue = generateReceiverOrNull(ktDefaultElement, extensionReceiver ?: dispatchReceiver)
             extensionReceiverValue = null
+            contextReceiverValues = emptyList()
```

Comment:
Question: what would happen if we have a typealias for a context class, and someone uses it as a class constructor?

Reply:
Thanks for noticing. [Commit.](https://github.com/JetBrains/kotlin/pull/4488/commits/cc705d22ae8438ec5d791c2daa6047a9b8bc6d48)